superset/security.py

```diff
@@ -85,6 +85,15 @@ def get_schema_perm(self, database, schema):
         if schema:
             return '[{}].[{}]'.format(database, schema)
 
+    def always_list_all_dashboards(self):
```

Comment:
I agree with your comment that we should defer to the security manager to answer said question, however I’m not sure we need these always_ methods.